Source/WebInspectorUI/UserInterface/Views/OverrideUserPreferencesPopover.js

```diff
@@ -171,6 +116,47 @@ WI.OverrideUserPreferencesPopover = class OverrideUserPreferencesPopover extends
         return contentElement;
     }
 
+    // Private
+
+    _createSelectElement({contentElement, id, label, preferenceName, preferenceValues, defaultValue})
```

Comment:
Nit: `_createSelectElement({contentElement, id, label, preferenceName, preferenceValues, defaultValue} = {})`

Reply:
IMO that's only necessary if these arguments are all optional.  if they're all always expected then we shouldn't have the `= {}`

tho frankly in that case then these should really be top-level parameters, not properties on a destructured object :/

Reply:
If there's more than 3-4 arguments, wrapping them in an object makes it easier to compose for the method call regardless of order. I'd keep it as is.